tools/monitoring/mysql/README.md

```diff
@@ -0,0 +1,33 @@
+# Monitoring Database
+
+The monitoring database is a MySQL instance on Cloud SQL. It stores the error logs for each prow job
+failures; and the alert status for all error patterns.
+
+## Database Connection With Cloud Shell
+
+Navigate to SQL in Google Cloud Platform. Click on the knative-monitoring instance. In the Overview tab,
+under Connect to this Instance section, click on Connect using Cloud Shell.
```

Comment:
I do not see " Connect using Cloud Shell" under "Connect to this Instance section". Maybe the UI changes?

I would suggest using quotations or _italic_ when quoting a name phrase

Reply:
Changed it to italics and added permission required in each section.